docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+
+
+networks:
+    my_app_net:
+        driver: bridge
+
+
+services:
+  mysql_service:
+    image: mysql_image:1.0
```

Comment:
The image name `mysql_image:1.0` suggests a custom image. Ensure that this image is built and available locally or on a Docker registry. If it's meant to be the official MySQL image, use `mysql:latest` or a specific version tag.